gen/gen.jl

```diff
@@ -1,52 +1,24 @@
 import Clang
+import HiGHS_jll
 
-const HIGHS_DIR = haskey(ENV, "HIGHS_PATH") ? ENV["HIGHS_PATH"] : joinpath(@__DIR__, "../deps/usr")
+const LIBHIGHS_HEADERS = [
+    joinpath(HiGHS_jll.artifact_dir, "include", "interfaces", "highs_c_api.h"),
+]
 
-const header_file = joinpath(HIGHS_DIR, "src/interfaces/highs_c_api.h")
-
-const LIB_HEADERS = [header_file]
-
-const ctx = Clang.DefaultContext()
+const GEN_DIR = joinpath(dirname(@__DIR__), "src", "wrapper")
```

Comment:
```suggestion
const GEN_DIR = joinpath(@__DIR__, "..", "src", "wrapper")
```
dirname of a dir just extracts the directory above, I find it more explicit with `..`